ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1313,6 +1313,52 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-ingress-internal
+- as: baremetal-compact-agent-ipv4-dhcp-day2-amd-mixarch-f14
```

Comment:
Do we need the test in 4.16 ?

Reply:
I think so, @aleskandro, @lwan-wanglin , should we move the jobs to 4.16?

Reply:
Considering we will postpone the feature to 4.16, I think we can move the jobs to 4.16

Reply:
Instead of move the job to 4.16, perhaps we can add a new job for 4.16 ( and also keep the 4.15 one ).  WDYT ?

Reply:
yes, that's ok, whether do we need to reduce frequency to f28 for 4.15? because of limited baremetal infra @aleskandro

Reply:
Yes, let's set f28 for 4.15 to get some early testing signal, but also open a story to remember of deleting it after GA on 4.16